jenkins_jobs/modules/triggers.py

```diff
@@ -511,6 +552,22 @@ def gerrit(registry, xml_parent, data):
         to Gerrit. (default false)
     :arg bool escape-quotes: escape quotes in the values of Gerrit change
         parameters (default true)
+    :arg dict build-cancellation-policy: If used, rules regarding
+        cancellation of builds can be set with this option when
+        patchsets of the same change comes in.
```

Comment:
I'd add that this setting overrides global server configuration.

Also if `build-cancellation-policy` is not present in YAML the global server configuration is used.